packages/react/utils/src/index.tsx

```diff
@@ -20,7 +20,7 @@ export function Show<T = boolean>(props: ShowProps<T>): JSX.Element | null {
 	useSignals();
 	const value =
 		typeof props.when === "function" ? props.when() : props.when.value;
-	if (!value) return props.fallback || null;
+	if (!value) return (props.fallback as JSX.Element) || null;
```

Comment:
This seems to be the correct approach for React's types? Weird: https://github.com/microsoft/TypeScript/issues/14729

If there's a better way lmk